scipy/special/special/specfun/specfun.h

```diff
@@ -5597,7 +5597,7 @@ void sphj(T x, int n, int *nm, T *sj, T *dj) {
         }
         sj[0] = 1.0;
         if (n > 0) {
-            dj[0] = 1.0 / 3.0;
+            dj[1] = 1.0 / 3.0;
```

Comment:
dj is zero indexed in the Fortran as well, so this should be a 1 here

https://github.com/scipy/scipy/blob/1d3a067c2ccd0a6efddeb3194163aa9a3879d26e/scipy/special/specfun/specfun.f#L9699-L9708